client/components/jetpack/jetpack-welcome-page/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Card } from '@automattic/components';
+import classnames from 'classnames';
+import JetpackLogo from 'calypso/components/jetpack-logo';
+import Main from 'calypso/components/main';
+
+import './style.scss';
+
+export type JetpackWelcomePageProps = {
+	description?: React.ReactNode;
+	footer?: React.ReactNode;
+	mainClassName?: string;
+	pageViewTracker?: React.ReactNode;
+	steps?: Array< React.ReactNode >;
+	title: React.ReactNode;
```

Comment:
We should keep necessary fields at the top for better visibility

Reply:
I don't know whether we "should", but I prefer to order the props alphabetically to make it easier to add new or find existing when there are too many.